recipes/inference_labels.ipynb

```diff
@@ -4,66 +4,88 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>use math notation ($ $) for y=m(x).</li><li>you can keep "What .... agreement:"  in the same paragraph</li><li><br></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>